Basic-Scripts/watermark_pdf.py

```diff
@@ -0,0 +1,26 @@
+'''
+This script helps to add water mark to any PDF's
+which can have single page to multiple pages using PyPDF2 module
+'''
+'''
+Intsall module using command 'pip install PyPDF2'
+'''
+import PyPDF2
+
+# Initializing PDF file reader object for Source PDF
+pdf_file_reader = PyPDF2.PdfFileReader(open('./PDF_Docs/give-and-take-BOOK.pdf','rb'))
```

Comment:
Instead of Using File Path Here initialize a global variable and document it for file path
Like: 
```
SOURCE_PDF = "<PROVIDE THE SOURCE PDF PATH HERE >"
WATERMARK_PDF = "< PROVIDE THE WATERMARK PDF PATH HERE >"
```

Reply:
Hi, I have made the changes as per your review!